x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/persistence/TransformInternalIndex.java

```diff
@@ -357,7 +357,7 @@ private static XContentBuilder addTransformCheckpointMappings(XContentBuilder bu
      * @throws IOException On write error
      */
     private static XContentBuilder addMetaInformation(XContentBuilder builder) throws IOException {
-        return builder.startObject("_meta").field("version", Version.CURRENT).endObject();
+        return builder.startObject("_meta").field("version", TransformConfigVersion.CURRENT).endObject();
```

Comment:
This one is an index mappings version, so needs to be part of https://github.com/elastic/elasticsearch/pull/97934, not this PR. (And it already is part of that other PR.)